<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImExport</class>
 <widget class="QWidget" name="ImExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="main">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0">
        <widget class="QLabel" name="hello">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>NEKO LIBRO CHÀO BẠN! BẠN MUỐN XUẤT/NHẬP HÀNG HÓA GÌ TRONG HÔM NAY?</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QToolButton" name="import_main">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>NHẬP HÀNG</string>
           </property>
           <property name="icon">
            <iconset resource="galeries.qrc">
             <normaloff>:/image/import_2.png</normaloff>:/image/import_2.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QToolButton" name="export_logs_main">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>LỊCH SỬ 
XUẤT HÀNG</string>
           </property>
           <property name="icon">
            <iconset resource="galeries.qrc">
             <normaloff>:/image/logs.png</normaloff>:/image/logs.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QToolButton" name="export_main">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>XUẤT HÀNG</string>
           </property>
           <property name="icon">
            <iconset resource="galeries.qrc">
             <normaloff>:/image/export.png</normaloff>:/image/export.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="import_logs_main">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>LỊCH SỬ 
NHẬP HÀNG</string>
           </property>
           <property name="icon">
            <iconset resource="galeries.qrc">
             <normaloff>:/image/logs.png</normaloff>:/image/logs.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
        <widget class="QLabel" name="logo_im_export">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="galeries.qrc">:/image/im_export.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="import_window">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="6" colspan="2">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>PHIẾU NHẬP HÀNG</string>
         </property>
        </widget>
       </item>
       <item row="15" column="11">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Số lượng</string>
         </property>
        </widget>
       </item>
       <item row="17" column="5" colspan="2">
        <widget class="QPushButton" name="import_logs_from_import">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Lịch sử nhập hàng</string>
         </property>
         <property name="icon">
          <iconset resource="galeries.qrc">
           <normaloff>:/image/logs.png</normaloff>:/image/logs.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="7" colspan="5">
        <widget class="QLabel" name="bill_num_import">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Ngày</string>
         </property>
        </widget>
       </item>
       <item row="13" column="11">
        <widget class="QPushButton" name="add_to_table_import">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Thêm</string>
         </property>
         <property name="icon">
          <iconset resource="galeries.qrc">
           <normaloff>:/image/add.png</normaloff>:/image/add.png</iconset>
         </property>
        </widget>
       </item>
       <item row="16" column="11">
        <widget class="QLabel" name="quantity">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="17" column="9">
        <widget class="QPushButton" name="minimum_import">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string> Cancel</string>
         </property>
         <property name="icon">
          <iconset resource="galeries.qrc">
           <normaloff>:/image/cancel.png</normaloff>:/image/cancel.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="7" colspan="5">
        <widget class="QLabel" name="date_import">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="17" column="10" colspan="2">
        <widget class="QPushButton" name="save_invoice_import">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Lưu phiếu</string>
         </property>
         <property name="icon">
          <iconset resource="galeries.qrc">
           <normaloff>:/image/save.png</normaloff>:/image/save.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="7" column="0" rowspan="10" colspan="11">
        <widget class="QTableWidget" name="import_books">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLabel" name="logo_import">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="galeries.qrc">:/image/import_2.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="14" column="11">
        <widget class="QPushButton" name="remove_from_table_import">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string> Xóa</string>
         </property>
         <property name="icon">
          <iconset resource="galeries.qrc">
           <normaloff>:/image/remove_2.png</normaloff>:/image/remove_2.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="17" column="0" colspan="5">
        <widget class="QPushButton" name="add_new_category_from_import">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string> Thêm danh mục sản phẩm</string>
         </property>
         <property name="icon">
          <iconset resource="galeries.qrc">
           <normaloff>:/image/category.png</normaloff>:/image/category.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="6">
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Mã đơn hàng</string>
         </property>
        </widget>
       </item>
       <item row="17" column="7" colspan="2">
        <widget class="QPushButton" name="back_to_main_from_import">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Quay lại</string>
         </property>
         <property name="icon">
          <iconset resource="galeries.qrc">
           <normaloff>:/image/back.png</normaloff>:/image/back.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="logo_search">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="galeries.qrc">:/image/search.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="10">
        <widget class="QLineEdit" name="search_product">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="export_window">
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>301</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>PHIẾU XUẤT HÀNG</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>20</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>50</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>20</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>50</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>100</y>
         <width>113</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>100</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>160</y>
         <width>256</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>180</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>230</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>500</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>500</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>500</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>510</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="import_logs_window">
      <widget class="QTableView" name="results_table">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>771</width>
         <height>261</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="start_date">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>110</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>10</y>
         <width>281</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>TRA CỨU LỊCH SỬ NHẬP HÀNG</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Từ ngày</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>80</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Đến ngày</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="end_date">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>80</y>
         <width>111</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="to_main">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>460</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Quay lại</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Kết quả tra cứu</string>
       </property>
      </widget>
      <widget class="QDialogButtonBox" name="to_main_2">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>460</y>
         <width>193</width>
         <height>29</height>
        </rect>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
       </property>
      </widget>
      <widget class="QPushButton" name="search">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>80</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Tìm kiếm</string>
       </property>
      </widget>
      <widget class="QPushButton" name="all_histories">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>120</y>
         <width>121</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Tất cả lịch sử</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="export_logs_window">
      <widget class="QLabel" name="label_33">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>10</y>
         <width>271</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>TRA CỨU LỊCH SỬ XUẤT HÀNG</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_34">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Từ ngày</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit_5">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>110</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_35">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>80</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Đến ngày</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit_6">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>80</y>
         <width>110</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_25">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>80</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Tìm kiếm</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_26">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>80</y>
         <width>101</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Tất cả lịch sử</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableView_7">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>150</y>
         <width>711</width>
         <height>281</height>
        </rect>
       </property>
      </widget>
      <widget class="QDialogButtonBox" name="to_main_6">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>450</y>
         <width>193</width>
         <height>29</height>
        </rect>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_37">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>120</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Kết quả tìm kiếm</string>
       </property>
      </widget>
      <widget class="QPushButton" name="to_main_4">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>460</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Quay lại</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="galeries.qrc"/>
 </resources>
 <connections/>
</ui>
